WordPress/Classes/Plugins/Views/PluginDetailsView.swift

```diff
@@ -413,4 +510,43 @@ private enum Strings {
         )
         return String(format: format, author)
     }
+
+    static let deletePluginTitle = NSLocalizedString(
```

Comment:
(nit) I'd got with single lines for these declarations. They are write-once and you rarely need to read/change them.